### ISSUE TITLE
Need to figure out a way to assign a collection _group_id to appropriate collection

### ISSUE THREAD

**Issue by aazaff** [azgs/azlibrary_database#5]
I don't have a clear index for the parent collect_group of each collection, but this WILL ultimately have to make its way into the database.

**Comment by aazaff** [comment]
On dev, I used the following workflow to at least assign a `collection_group_id` to the digital geologic maps.

1. I took the URLs for all DGMs listed in the [geologic map tracking repo](https://github.com/azgs/geologic_map_tracking/blob/master/intern_resources/2018NIDRepoUpdate%20(3).xlsx)
2. Created a temporary table in SQL
3. Then used the following SQL

````SQL
UPDATE collections SET collection_group_id=1 WHERE azgs_old_url IN (SELECT * FROM temp_maps);
````

I'm not closing this issue because this is obviously not a production level solution...

**Comment by aazaff** [comment]
Sent @NoisyFlowers the attached lookup table. Should be sufficient for him to update the azgs.json. I got to this information by accessing the **v_dlio_metadata_list** view in the drupal site.

[metadata.csv.zip](https://github.com/azgs/azlibrary_database/files/2852309/metadata.csv.zip)

**Comment by NoisyFlowers** [comment]
I've incorporated the lookup table into the schema setup logic. I've added logic in azlibConvert to use this new information in assign collection group to old collections prior to import. 

azlibConvert now accepts two databases as parameters, and old and a new. The old is mined for informal_name and ua_library. The new is mined for the collection group info. Management of these two db's, from parameter processing through query and close, works, but error reporting might be a little janky. I'm not sure how much time to spend cleaning this up, as I don't know how much use azlibConvert will see after initial import. Perhaps this is worth a separate, low priority, issue?

On dev, there is a db called tmplibrary that is fully populated and includes these changes. It looks good to me but should probably be further compared to the existing azlibrary on that machine.
